restcomm/restcomm.mscontrol.mms/src/main/java/org/restcomm/connect/mscontrol/mms/MgcpMediaGroup.java

```diff
@@ -253,8 +255,12 @@ protected void notification(final Object message) {
         } else {
             event = new MediaGroupResponse<>(response.cause(), response.error());
         }
-        if (originator != null)
+        if (originator != null) {
             this.originator.tell(event, self);
+            if (recording) {
+                recording = false;
```

Comment:
set to false only if originator is not null? is this right? shouldnt we set to false in any case since we have the ivrresponse?